docs/_tutorials/reference.md

```diff
@@ -114,7 +114,9 @@ App options are passed into the `App` constructor.
 | `logLevel` | Option to control how much or what kind of information is logged. The `LogLevel` export contains the possible levels–in order of most to least information: `DEBUG`, `INFO`, `WARN`, and `ERROR`. By default, `logLevel` is set to `INFO`. More information on logging can be found [in the documentation](/bolt-js/concepts#logging). |
 | `extendedErrorHandler` | Option that accepts a `boolean` value. When set to `true`, the global error handler is passed an object with additional request context. Available from version 3.8.0, defaults to `false`. More information on advanced error handling can be found [in the documentation](/bolt-js/concepts#error-handling). |
 | `ignoreSelf` | `boolean` to enable a middleware function that ignores any messages coming from your app. Requires a `botId`. Defaults to `true`.  |
-| `clientOptions.slackApiUrl` | Allows setting a custom endpoint for the Slack API. Used most often for testing. | 
+| `clientOptions.slackApiUrl` | Allows setting a custom endpoint for the Slack API. Used most often for testing. |
+| `socketMode` | Option that accepts a `boolean` value. When set to `true` the app is started in socket mode, i.e. it allows your app to connect and receive data from Slack via a WebSocket connection. Defaults to `false`. See [socket mode documentation](/bolt-js/concepts#socket-mode) for further information.
```

Comment:
```suggestion
| `socketMode` | Option that accepts a `boolean` value. When set to `true` the app is started in [Socket Mode](/bolt-js/concepts#socket-mode), i.e. it allows your app to connect and receive data from Slack via a WebSocket connection. Defaults to `false`. See [Socket Mode documentation](/bolt-js/concepts#socket-mode) for further information.
```

Reply:
I like the idea to directly link to the socke mode documentation when mentioning the mode for the first time. However, I will remove the last sentence because this becomes redundant information.